src/prop-types.js

```diff
@@ -92,8 +92,15 @@ const borderWidth = {
   borderLeft: PropTypes.bool
 }
 
+const borderColor = {
```

Comment:
Thanks for this! This will also need to be exported from the module

Reply:
Good catch, @jxnblk. I took care of it.